doc/operations/start_network.rst

```diff
@@ -138,54 +160,28 @@ Using a Configuration File
-
-.. note:: After the network is open to users, members can still issue governance transactions to CCF (for example, adding new users or additional members to the consortium). See :doc:`/governance/index` for more information about member governance.
-
-The following diagram summarises the steps required to bootstrap a CCF network:
-
-.. mermaid::
+Virtual Mode
+------------
 
-    sequenceDiagram
-        participant Operators
-        participant Members
-        participant Users
-        participant Node 0
-        participant Node 1
+To start a CCF node in `virtual` mode, operators should use the `virtual` variants of the ``cchost`` binary and enclave application:
 
-        Operators->>+Node 0: cchost start --rpc-address=ip0:port0
-        Node 0-->>Operators: Network Certificate
-        Note over Node 0: Part Of Network
+.. code-block:: bash
 
-        Operators->>+Node 1: cchost join --network-cert-file=Network Certificate --target-rpc-address=ip0:port0
+    $ cchost.virtual --enclave-file /path/to/virtual_enclave_library [args]
 
-        Node 1->>+Node 0: Join network (over TLS)
-        Node 0-->>Node 1: Network Secrets (over TLS)
+.. warning:: Nodes started in virtual mode provide no security guarantees. They should never be used for production purposes.
 
-        Note over Node 1: Part Of Network
+Signature Interval
+------------------
 
-        loop Governance transactions (e.g. adding a user)
-            Members->>+Node 0: HTTP Request (any node)
-            Node 0-->>Members: HTTP Response (any node)
-        end
+Transaction commit latency in a CCF network is primarily a function of signature frequency. A network emitting signatures more frequently will be able to commit transactions faster, but will spend a larger proportion of its execution resources creating and verifying signatures. Setting signature frequency is a trade-off between transaction latency and throughput.
 
-        Members->>+Node 0: Propose to open network (any node)
-        Members->>+Node 0: Vote to open network (any node)
-        Note over Node 0, Node 1: Proposal accepted, CCF open to users
+Two options are provided to that end:
 
+- ``--sig-tx-interval``: number of transactions between two signatures
+- ``--sig-ms-interval``: time in milliseconds between two signatures
 
-        loop Business transactions
-            Users->>+Node 0: HTTP Request (any node)
-            Node 0-->>Users: HTTP Response (any node)
-        end
+.. note:: These options specify the intervals at which the generation of signature transactions is `triggered`. However, because of the parallel execution of transactions, it is possible that signature transactions are recorded in the ledger at a slightly higher interval than the specified values.
```

Comment:
```suggestion
.. note:: These options specify the intervals at which the generation of signature transactions is `triggered`. However, because of the parallel execution of transactions, the actual intervals between signature transactions may be slightly larger.
```

I think "at a slightly higher interval" is odd wording, prefer _larger_ interval. Reworded to clarify this.